projects/igniteui-angular/src/lib/core/styles/components/grid/_excel-filtering-theme.scss

```diff
@@ -222,19 +220,24 @@
         }
     }
 
-    %grid-excel-move igx-buttongroup {
-        @include if-rtl() {
+    %grid-excel-move .igx-button-group {
+        [dir='rtl'] & {
             flex-direction: row-reverse;
 
+            igx-icon,
             [igxButton] {
-                flex-direction: row-reverse;
+                direction: ltr;
+            }
+        }
+    }
 
-                span,
-                igx-icon {
-                    margin: initial;
-                }
+    %grid-excel-sort .igx-button-group {
+        [dir='rtl'] & {
+            flex-direction: row-reverse;
 
-                @include icon-w-margin(rem(8px), $right);
```

Comment:
Something here became wrong and the excel-filtering doesn't look the same as before.

This is how it looked like before:
<img width="551" alt="Screenshot 2022-04-13 at 14 17 41" src="https://user-images.githubusercontent.com/45598235/163169565-883d009d-0369-4ad9-9d6c-7b781a6bea48.png">


This is how it looks like now: 
<img width="496" alt="Screenshot 2022-04-13 at 14 17 34" src="https://user-images.githubusercontent.com/45598235/163169558-b5910e01-43ca-4da9-993f-b857ba9bf22c.png">

Reply:
Correct, now it looks like it should.